api/src/routes/api.ts

```diff
@@ -65,8 +66,20 @@ router.get("/:category/:job/:image/:size", async function(req, res, next) {
     let size: string = req.params.size;
     let job = getJobFromRequest(req);
     let deriv = await job.getImage(image).derivative(legalSizes[size] ?? "THUMBNAIL");
-    console.log(deriv);
     res.sendFile(deriv);
 });
 
+router.delete("/:category/:job/:image/*"), async function(req, res, next) {
+    //return ImageFile.delete();
+    let image: string = req.params.image;
```

Comment:
We should sanitize/validate this parameter (along with all the others!). Not sure if there's a way to apply constraints to route parameters through the route definition, or if we need to add logic to do the work... but one way or another, that's a detail we should handle soon, since it's a security thing.